RA Scripts/Legend of Zelda, The The Minish Cap.rascript

```diff
@@ -1110,61 +1119,65 @@ GenerateSaveProtectedAchievementFromConditions({
 achievement(
     title = "Bested Big Green Chuchu [m]", description = "Defeat the Big Green Chuchu without taking damage", points = 10,
     id = 2078, badge = "155845", published = "8/5/2013 5:15:22 AM", modified = "8/5/2013 5:31:38 AM",
-    trigger = once(byte(0x000BF4) == 0x48) && once(byte(0x000BF5) == 0x17) && never(byte(0x000BF5) == 0x0B) && 
-              word(0x000BF4) == 0x0049 && never(byte(0x00AAEA) < prev(byte(0x00AAEA))) && once(byte(0x00188D) == 0xFF) && trigger_when(byte(0x00188D) == 0x00) && 
+    trigger = once(ScreenIdByte() == 0x48) && once(byte(0x000BF5) == 0x17) && never(byte(0x000BF5) == 0x0B) && 
+              ScreenIdWord() == 0x0049 && never(byte(0x00AAEA) < prev(byte(0x00AAEA))) && once(byte(0x00188D) == 0xFF) && trigger_when(byte(0x00188D) == 0x00) && 
               never(bit0(0x00AB42) < prev(bit0(0x00AB42))) && bit0(0x00AB42) == 0x1
 )
 
+firstTimeGameBeaten = bit0(0x00AA46) > prev(bit0(0x00AA46))
```

Comment:
Let's make this a function instead of a variable. (As such, its name should be in PascalCase instead of camelCase, as well.)

Reply:
Can do.